opba-api-security/src/main/java/de/adorsys/opba/api/security/internal/service/TokenBasedAuthService.java

```diff
@@ -16,6 +17,7 @@
 import java.time.ZonedDateTime;
 import java.util.Date;
 
+@Slf4j
```

Comment:
Slf4j doesn't seem to be used in this class

Reply:
yep, no more